app/controllers/users_controller.rb

```diff
@@ -17,7 +17,7 @@ def destroy
 
   def show
     @title = "Most recent predictions by #{@user}"
-    @predictions = PredictionFilter.filter(@user, current_user, params[:filter], params[:page])
```

Comment:
The functionality of `PredictionFilter` seemed like a proper subset of the `PredictionsQuery`, so I ended up deleting the former entirely, but adding a test case demonstrating that `PredictionsQuery` filters by status properly.

Reply:
Seems fine